cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -143,8 +143,8 @@ def check_ftp(appliance, ftp, server_name, server_zone_id, check_ansible_logs=Fa
         zip_files = ftp.filesystem.search(re.compile(
             r"^.*{}{}[.]zip$".format(server_string, date_group)), directories=False)
         assert zip_files, "No logs found!"
-        # Collection of Models and Dialogs introduced in 5.10
-        if not BZ(1656318, forced_streams=["5.10"]).blocks:
+        # Collection of Models and Dialogs introduced in 5.10 but it work only in 5.11 (BZ 1656318)
+        if appliance.version >= "5.11" and not BZ(1706989).blocks:
```

Comment:
Is the appliance version check necessary? Shouldn't this BZ block only for 5.11?

Reply:
@john-dupuy there're 2 different BZs, one is fixed for 5.11 and was not cloned to 5.11 :(